core-util/FunctionPointerBase.h

```diff
@@ -54,6 +54,27 @@ class FunctionPointerBase {
         memset(_member, 0, sizeof(_member));
     }
 
+    /**
+     * Calls the member pointed to by object::member or (function)object
+     * @param arg
+     * @return
+     */
+    inline R call(void* arg) {
+#ifndef YOTTA_CFG_UTIL_FUNCTIONPOINTER_DISABLE_NULL_CHECK
```

Comment:
Not sure, but it's not original to this PR, so any concerns about this should be filed in an issue.